### DISCUSSION TITLE
Do `schedulers` have to be configured differently in `0.8.0`

### DISCUSSION THREAD

**Discussion by devraj** [taskiq-python/taskiq#170] [Q&A] [answered]
First of all thank you for releasing `0.8.0` which is very timely with `FastAPI` releasing their compatibility with `pydantic2`. Upon upgrading I am getting the following error when launching the scheduler.

```
Imported scheduler is not a subclass of TaskiqScheduler.
```

Nothing has changed on our side and the `scheduler` configuration matches what is specified in the documentation:

```python
scheduler = TaskiqScheduler(
    broker=broker,
    sources=[
        LabelScheduleSource(broker)
    ],
)
```

and it's runs as following in a container

```
command: [ "taskiq", "scheduler", "${PROJ_NAME}.broker:scheduler" ]
```

all referenced code is available on `main` branch of [lab-python-server](https://github.com/anomaly/lab-python-server) 

Appreciate any pointers. 🙏

**Comment by s3rius** [comment]
I'll take a look.

**Reply by s3rius** [reply, marked as answer]
Hi, @devraj! Thanks for finding it out quickly. I managed to fastfix the issue in https://github.com/taskiq-python/taskiq/releases/tag/0.8.1. 

Sorry for inconvenience.
